source/install/troubleshooting.rst

```diff
@@ -19,9 +19,40 @@ General Troubleshooting
 -----------------------
 Some of these suggestions can be done directly, and others may need consultation from your network administrator.
 
-- Take a look at the logs (``mattermost.log`` and NGINX logs) for errors.
-- You can also search the error messages online - existing solutions can often be applied.
-- Open **System Console > General > Logging** and set File Log Level to **DEBUG**. Make sure to revert to **INFO** after troubleshooting to save disk space.
+Take a Look at the Logs
+~~~~~~~~~~~~~~~~~~~~~~~
+
+You can access logs for Mattermost and use them for troubleshooting. These steps assume that you have System Admin permissions. 
```

Comment:
```suggestion
You can access logs for Mattermost and use them for troubleshooting. These steps assume that you have `System Admin permissions <https://docs.mattermost.com/help/getting-started/managing-members.html#system-admin>`_. 
```

Reply:
For someone new, they may not know what permissions System Admin has, so we can link to a doc that offers this information.